lib/pal/WorkerThread.cpp

```diff
@@ -182,69 +182,74 @@ namespace PAL_NS_BEGIN {
             LOG_INFO("Running thread %u", std::this_thread::get_id());
 
             for (;;) {
-                std::unique_ptr<MAT::Task> item = nullptr;
-                wakeupCount++;
-                unsigned nextTimerInMs = MAX_FUTURE_DELTA_MS;
+                #ifdef MacOS
```

Comment:
This should be applicable to both Mac and iOS

Reply:
`#ifdef APPLE` I believe. Otherwise, you can do or as well with `|` operator.

Reply:
`ifdef __APPLE__` should suffice for both Mac and iOS.

Reply:
Fixed the macro